src/test/integration/web_interface/rest/test_rest_binary.py

```diff
@@ -1,29 +1,35 @@
+# pylint: disable=attribute-defined-outside-init
+
 from base64 import standard_b64encode
 from multiprocessing import Queue
 
 from intercom.back_end_binding import InterComBackEndBinding
 from storage.db_interface_backend import BackEndDbInterface
-from test.common_helper import create_test_firmware
+from test.common_helper import create_test_firmware, put_binary_for_binary_service
 from test.integration.intercom import test_backend_scheduler
 from test.integration.web_interface.rest.base import RestTestBase
 
 
 class TestRestDownload(RestTestBase):
 
-    def setUp(self):
-        super().setUp()
+    def setup(self):
+        super().setup()
         self.db_interface = BackEndDbInterface(self.config)
         self.test_queue = Queue()
 
-    def tearDown(self):
+    def teardown(self):
         self.test_queue.close()
         self.db_interface.shutdown()
-        super().tearDown()
+        super().teardown()
 
     def test_rest_download_valid(self):
-        backend_binding = InterComBackEndBinding(self.config, analysis_service=test_backend_scheduler.AnalysisServiceMock(), compare_service=test_backend_scheduler.ServiceMock(self.test_queue), unpacking_service=test_backend_scheduler.ServiceMock(self.test_queue))
-
+        backend_binding = InterComBackEndBinding(
+            self.config, analysis_service=test_backend_scheduler.AnalysisServiceMock(),
```

Comment:
Ignore if you like, but I really like to see all named parameters instead of positional / named mix. Also I'd put config on it's own line, but that's just personal preference.

Reply:
changed